examples/connext_dds/real_time_wan_transport/c++/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# (c) 2021 Copyright, Real-Time Innovations, Inc. All rights reserved.
+# No duplications, whole or partial, manual or electronic, may be made
+# without express written permission.  Any such copies, or revisions thereof,
+# must display this notice unaltered.
+# This code contains trade secrets of Real-Time Innovations, Inc.
+cmake_minimum_required(VERSION 3.11)
+project(rtiexamples-real-time-wan-transport)
+set(CMAKE_MODULE_PATH
+    ${CMAKE_MODULE_PATH}
+    "${CMAKE_CURRENT_SOURCE_DIR}/../../../../resources/cmake"
+)
+
+# Include ConnextDdsAddExample.cmake from resources/cmake
+include(ConnextDdsAddExample)
+
+connextdds_add_example(
+    IDL "real_time_wan_transport"
+    LANG "C++"
+    REQUIRE_SECURITY TRUE
+)
+
+configure_file(
```

Comment:
Not sure what is the best way to deploy the **CLOUD_DISCOVERY_SERVICE.xml** configuration. I took this from the way we deploy Routing Service files. However, another option is to assume that some of the examples may ship with some IS configuration and in such case we could integrate the copy inside connextdds_add_example